src/stravalib/model.py

```diff
@@ -717,32 +716,33 @@ class ActivityComment(Comment):
     athlete: Optional[SummaryAthlete] = None
 
 
-class ActivityPhotoPrimary(Primary):
-    """
-    Represents the primary photo for an activity.
-
-    Attributes
-    ----------
-    use_primary_photo : bool, optional
-        Indicates whether the photo is used as the primary photo.
+class PhotosSummary_primary(BaseModel):
+    """A object to store data about the primary photo of a Strava activity
 
     Notes
     -----
-    Attributes for activity photos are currently undocumented.
+    https://developers.strava.com/docs/reference/#api-models-PhotosSummary_primary
+    This object is nor returned in strava_model but it is documented in the
+    Strava API V3 spec. The Spec documentation deviates from what is actually
+    returned. This model represents what is actually returned
     """
 
-    use_primary_photo: Optional[bool] = None
+    unique_id: str | None = None
+    src: int | None = None
+    media_type: int | None = None
+    urls: dict[str, AnyHttpUrl] | None = None
 
 
-class ActivityPhotoMeta(PhotosSummary):
+class PhotosSummary(strava_model.PhotosSummary):
```

Comment:
similar to the above. NOTE that the spec doesn't align with what's actually returned but this is close